src/Microsoft.DotNet.Darc/src/Darc/Options/AddBuildToChannelCommandLineOptions.cs

```diff
@@ -40,6 +40,9 @@ internal class AddBuildToChannelCommandLineOptions : CommandLineOptions
         [Option("sdl-validation-continue-on-error", HelpText = "Ignore SDL validation errors.")]
         public string SDLValidationContinueOnError { get; set; }
 
+        [Option("bypass-promotion-pipeline", HelpText = "Add the build to the channel without publishing assets to the channel's feeds.")]
+        public bool BypassPromotionPipeline { get; set; }
```

Comment:
This can probably use a rename after the Help text change. SkipAssetPublishing or something similar?

Reply:
Makes sense. done.